src/ready/apis/train_mobious.py

```diff
@@ -220,25 +225,40 @@ def main(args):
     # .get() returns None if operation is None or config arg not valid
     transform_arg = transform_map.get(config.transforms.transform_operation, None)
     target_transform_arg = transform_map.get(config.transforms.target_transform_operation, None)
-
+    
+
+    # Split datapath into trainpath, validation+path, test_path
+    # Create full mobious dataset first, then split into respective sets.
+    # Make sure to change dataset fed into trainloader
+    # use .random_split()
+    # Put validation into training loop
+    # Finally, evaluate with the test_set
+    
     ## Length 5; github_data_path
     ## Length 1143;  data_path
-    trainset = MobiousDataset(
+    full_dataset = MobiousDataset(
         data_path, transform=transform_arg ,target_transform=target_transform_arg
         )
+    
+    SEED = 42
```

Comment:
Can add details in the doctring why we are using SEED 42? Should we use a specific SEED value? does it work with any value?